cmd/cstor-pool-mgmt/volumereplica/volumereplica.go

```diff
@@ -108,7 +108,7 @@ func CheckValidVolumeReplica(cVR *apis.CStorVolumeReplica) error {
 }
 
 // CreateVolumeReplica creates cStor replica(zfs volumes).
-func CreateVolumeReplica(cStorVolumeReplica *apis.CStorVolumeReplica, fullVolName string) error {
+func CreateVolumeReplica(cStorVolumeReplica *apis.CStorVolumeReplica, fullVolName string, quorum bool) error {
```

Comment:
To avoid passing more arguments to function need to maintain some annotations in CVR which will be decide about quorumValue as it mentioned in before comment.